src/dreyfus_config.erl

```diff
@@ -0,0 +1,11 @@
+ -module(dreyfus_config).
+
+ -export([data/0, get/1]).
+
+data() ->
+    Data = config:get("dreyfus", "black_list", []),
```

Comment:
I thought about it over the weekend and think that we shouldn't do that.
This configuration gives as a number of complications:
1. the operators  have to use API and cannot use regular config
   - which means that tooling wouldn't be able to use `_config` endpoint
   - complex term would be stored in the config
   - there is a possibility that operator would use config by mistake and overwrite the rules.
2. it requires complex `in_black_list/3` and `add_bl_element/1` functions

It would simplify things a lot if we change the way we configure the blacklist somehow.

Reply:
@iilyak :  @jiangphcn  had the same concern with the complex functions. What do you think about moving the complex add/remove over into snippet? We would still keep ```in_black_list/3``` but it would not be called by any operators. They can obviously still update the config manually by doing ```rpc:call/3``` or ```config:set/3```.

Reply:
@tonysun83 I think that it would be better to move the complex add/remove to snippet. Operators are more familiar with snippet than dbcore codes. Thus, they may easily use the snippet to add/remove entry from blacklist.

Reply:
@jiangphcn : i've moved the functions over to test utils. They will also be added to snippet

Reply:
@tonysun83: What if we would replace a single configuration key with a section. every rule would get own key. For example:
```
[dreyfux.blacklist]
mydb3/_design/myddocid3/myindexname3 = true
```
The code change to this PR would be 2-3 lines. However it wouldn't be possible for operator to make a mistake. Also there is a possibility that the direct remsh access would go away (security and compliance requirements) so snippets wouldn't be a solution.